brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1440 2022 Theme compatibility - Frontend styling issue resolved.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1440

### Screenshots


### Types of changes




- Improvement.

### How has this been tested?

- Activate the 2022 theme.
- Go to theme builder.
- Add a block, add styling to the block, and Save the changes.
- Visit frontend styling should apply on front.

### Checklist:
